lib/payloadAction.ts

```diff
@@ -44,14 +45,12 @@ export const  handlePayloadActions = async (app: IApp, read: IRead,  modify: IMo
                     const { visitor: { livechatData } } = livechatRoom;
                     await sendWelcomeEventToDialogFlow(app, read, modify, persistence, http, rid, visitorToken, livechatData);
                 } else if (actionName === ActionIds.SET_TIMEOUT) {
-
                     const event = { name: params.eventName, languageCode: LanguageCode.EN, parameters: {} };
```

Comment:
I know this is not part of this PR. I think we forgot to add language change here. For CX it should first use from persistent storage, right? We can fix it separately.